one-pager/banner/main.py

```
@@ -0,0 +1,71 @@
+    try:
+        client = genai.Client()
+        response = client.models.generate_images(
+            model='imagen-4.0-generate-preview-06-06',
+            prompt=f"Brief description: '{prompt}' Make a banner for it, include the startup name, think of a Linkedin banner.",
+            config=types.GenerateImagesConfig(
+                number_of_images= 1,
+                aspect_ratio= "16:9"
+            )
+        )
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)